src/platform/silabs/platformAbstraction/WiseMcuSpam.cpp

```diff
@@ -20,11 +20,16 @@
 #include <FreeRTOS.h>
 #include <task.h>
 
+#if SILABS_LOG_ENABLED
+#include "silabs_utils.h"
+#endif
+
 // TODO add includes ?
 extern "C" void RSI_Board_LED_Set(int, bool);
 extern "C" void RSI_Board_LED_Toggle(int);
 extern "C" void RSI_Wakeupsw_config(void);
 extern "C" void RSI_Wakeupsw_config_gpio0(void);
+extern "C" void sl_platform_init(void);
```

Comment:
why does it have to be extern? you can include the header in here for the sl_platform

Reply:
done